### PR TITLE
Remove VALID_USERNAME regex in utils.py

### PR DESCRIPTION
**Pull request by sindhus**

### CODE REVIEW

File: lastuser_core/models/user.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def username(self, value):
         elif self.is_valid_username(value):
             self._username = value
 
-    # Alias name to username
+    # Alias username to name
```
Comment (jace) [inline lastuser_core/models/user.py]: This uses `alias` in the verb form, which implies "name" is an alias for "username". The old comment was correct. https://en.wiktionary.org/wiki/alias#Verb

File: lastuser_core/utils.py [hunk 2]
```diff
@@ -33,7 +33,6 @@ def make_redirect_url(url, use_fragment=False, **params):
         urlparts[3] = make_query_string(queryparts)
     return urlparse.urlunsplit(urlparts)
 
-
```
Comment (jace) [inline lastuser_core/utils.py]: Top level functions and classes should have two blank lines separating them. Restore this.

File: lastuser_core/models/user.py (hunk 1 above)
Comment (sindhus) [inline lastuser_core/models/user.py]: Oh, ok, will remove this commit.

File: lastuser_core/utils.py [hunk 3]
```diff
@@ -41,6 +40,8 @@ def strip_phone(candidate):
 def valid_phone(candidate):
     return not PHONE_VALID_RE.search(candidate) is None
 
+def valid_username(candidate):
+    return not USERNAME_VALID_RE.search(candidate) is None
```
Comment (jace) [inline lastuser_core/utils.py]: Where was this introduced from? Also, extra blank line required above and below.

Comment (sindhus) [inline lastuser_core/utils.py]: The regex existed but there was no corresponding function to check for validity of username. So I added the function.

I just checked valid_username() is being used in lastuser is from coaster.utils. Nowhere else in code the regex is being used. So I guess there is no need for the regex?

Comment (jace) [inline lastuser_core/utils.py]: Yes. The function moved to coaster.utils but the regex was missed. We can remove it.

**Comment by sindhus** [comment]
Can you merge this @jace ?

**Pull request merged**